Dockerfile

```diff
@@ -40,6 +40,7 @@ WORKDIR /app
 
 COPY --chown=${UID}:${GID} --from=maven target/value-added-service-*.jar app.jar
 
+
 # Adding wget for the health check
 RUN apk --no-cache add wget
```

Comment:
Please also remove this one as it also modifies the base image. I would suggest using curl instead like [here](https://github.com/eclipse-tractusx/item-relationship-service/blob/9f39b0dec2586423873d53081db3e1bfc9f0c529/Dockerfile#L67).